lib/libutils/isoc/bget.c

```diff
@@ -1497,54 +1493,65 @@ static bufsize blimit(bs)
     return bs;
 }
 
-int main()
+int bget_main_test(void *(*malloc_func)(size_t), void (*free_func)(void *))
```

Comment:
Please mention that this function is added in the commit description. For example: "When enabled, the test entry point is bget_main_test() in libutils."

Reply:
OK